### ISSUE TITLE
Weird style transformation on multiple selectors + :global() + line break

### ISSUE THREAD

**Issue by nkzawa** [zeit/styled-jsx#411]
When transpiled the following css:

```css
.foo :global(path),
.bar
   :global(path) {
   stroke: #067df7;
 }
```

Generated css is:

```css
.jsx-123{.foo :global(path),.bar;}path{stroke:#067df7;}
```

It's no problem when css is like:

```css
// no line break
.foo :global(path),
.bar :global(path) {
  stroke: #067df7;
}
```

**Comment by giuseppeg** [comment]
@nkzawa looks like a bug in Stylis. I am able to reproduce on https://stylis.js.org 

cc @thysultan

**Comment by thysultan** [comment]
Works with the following config(required semicolons): `stylis.set({semicolon: true, cascade: false})`. I'll see if i can improve the no-semicolon option to also handle this. Can you open a corresponding issue on the stylis repo.

**Comment by nkzawa** [comment]
Thanks, created the issue on stylis: https://github.com/thysultan/stylis.js/issues/97

**Comment by nkzawa** [comment]
@thysultan thanks for the fix!

@giuseppeg https://github.com/zeit/styled-jsx/pull/413 can we have a release ? We're having the problem on our homepage.

**Comment by giuseppeg** [comment]
@nkzawa just released `2.2.5`
